README.md

```diff
@@ -37,11 +37,12 @@ pip install -q -U bitsandbytes
 pip install -q -U git+https://github.com/huggingface/transformers.git
 pip install -q -U git+https://github.com/huggingface/peft.git
 pip install -q -U git+https://github.com/huggingface/accelerate.git
+pip install -U datasets evaluate scipy nltk
```

Comment:
I think these might change. It would be better to put these in a requirements.txt file. Also, note that we removed the nltk dependency.